collections/_article/progress-report-web-export-in-4-3.md

```diff
@@ -0,0 +1,281 @@
+
+If you encounter any issue while testing my PR or while testing out the single-threaded samples builds, please report
+them [here](https://github.com/godotengine/godot/pull/85939).
+
+Finally, if you still intend to continue using threads on the Web platform, you can! And the next feature should aid you
+setup your game online.
+
+## Easy <abbr title="Cross-Origin-Opener-Policy">COOP</abbr>-<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> <abbr title="Progressive Web App">PWA</abbr> for threaded builds
+With this new feature, if you still need to export your game with thread support, you'll are able to activate the necessary
+<abbr title="Cross-Origin-Opener-Policy">COOP</abbr>/<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> headers
+without having to modifiy the server's header responses. You'll need to export your game as a Progressive Web App though.
+
+![Ensure Cross-Origin Isolation Headers export option](/storage/blog/progress-report-web-export-in-4-3/ensure-cross-origin-isolation-headers.webp)
+
+This is because this export option setup the Service Worker, included with the <abbr title="Progressive Web App">PWA</abbr>
+build, to intercept the requests from the server and inject those missing
+<abbr title="Cross-Origin-Opener-Policy">COOP</abbr>-<abbr title="Cross-Origin-Embedder-Policy">COEP</abbr> headers.
+
+## Boot splash screen
+Just before the beta feature freeze deadline, [@patwork](https://github.com/patwork), a new
+[#web](https://chat.godotengine.org/channel/web) contributor,
+include [some neat features](https://github.com/godotengine/godot/pull/91128):
+
+He cleaned up the game page code and he updated the progress bar to use the semantic (and very useful) HTML `<progress>`
+tag.
+
+But, mainly and from now on, he made the boot splash screen set in the project settings to be displayed on the screen
+while the engine and the assets are getting loaded by the browser.
+
+![New Godot Web loading screen](/storage/blog/progress-report-web-export-in-4-3/loading-screen.gif)
+
+If no boot splash screen has been set, the classic Godot boot splash will be seen, as it is currently displayed on
+other platforms.
+
+# Next steps
+Don't worry, these changes only mark the beginning for the Godot Engine to truly offer game developers a great and
+reliable way to create Web games.
+
+So work will be done to finalize and stabilize the samples playback for the Web. In the future, the node chain system
+should be powerful enough to support audio effects on buses, so we intend to work on this too.
+
+Based on the feedback we received at the <abbr title="Game Developers Conference">GDC</abbr> from partners and friends,
+we know that we need a way to reduce the size of our exports. Currently, the 4.3 release Web build .wasm is around 50MB.
+This is the engine only, without any assets. This may feel small nowadays, especially with gigabytes we're used to see
+on desktop and console games, but it is way too large for Web "instant games".
```

Comment:
I would avoid mentioning instant games at all. For the most part users aren't asking for instant game support, they just want decent support to run their games on itch.io. instant games are a whole other can of worms

```suggestion
on desktop and console games, but it is way too large for Web games.
```